src/plugins/autoplay/autoplay.js

```diff
@@ -36,15 +39,25 @@
             autoplayDefault = util.toNumber( data.autoplay, 0 );
         }
 
+        if ( data.userSkipAutoplay ) {
```

Comment:
Naming convention: You must call the attribute `data-autoplay-something=""`

See https://github.com/impress/impress.js/blob/master/src/plugins/README.md#structure-naming-and-policy